docs/data/base/components/button/UnstyledButtonIntroduction/css/index.js

```diff
@@ -7,8 +7,8 @@ export default function UnstyledButtonsIntroduction() {
   return (
     <React.Fragment>
       <Stack spacing={2} direction="row">
-        <Button className="CustomButton">Button</Button>
-        <Button className="CustomButton" disabled>
+        <Button className="IntroductionButton">Button</Button>
+        <Button className="IntroductionButton" disabled>
```

Comment:
This class name was shared between two demos, it was annoying when in the dev tools to have duplication of styles.